src/sentry/incidents/models/alert_rule.py

```diff
@@ -162,14 +163,21 @@ def conditionally_subscribe_project_to_alert_rules(
                             "origin": origin,
                             "query_extra": query_extra,
                             "condition": activation_condition,
+                            "restrict_to_window": restrict_to_window,
                         },
                     )
-                    # attempt to subscribe the alert rule
+                    now = timezone.now()
+                    window_delta = timedelta(seconds=alert_rule.snuba_query.time_window)
```

Comment:
Snuba requires both a start `timestamp > ...` and an end `timestamp < ...` in the query

In order to calculate the end timestamp, we need the `time_window_ from the `snuba_query`
So i need to construct the query_extra here